bastion-userdata.tmpl

```diff
@@ -143,6 +143,29 @@ fi
 # which are shell commands to be executed to create and configure the users.
 ${additional_user_templates}
 
+# run the github-users.sh script from s3
+aws s3 --region $${EC2_REGION} cp s3://${infrastructure_bucket}/${infrastructure_bucket_github_users_script_key} /usr/local/bin/github-users
```

Comment:
Related to my comment in `inputs.tf`, this probably should also include The `infrastructure_bucket_bastion_key` prefix.